airsonic-main/pom.xml

```diff
@@ -164,6 +164,12 @@
             <artifactId>commons-collections</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-collections4</artifactId>
```

Comment:
I'm not sure that we want to have two versions of the same package. Can't you use `commons-collections` instead?

Reply:
CircularFifoQueue is just available in commons-collections4.
I left both packages due to incompatibility with the CollectionsUtils class.
Do you want to keep a single package? I can do that. I would rather keep collections4 which is more powerful.

Reply:
Keep the latest commons-collection one if possible then :)

Reply:
@jvoisin should be good to go ;)